libs/s25main/world/MapLoader.cpp

```diff
@@ -98,7 +98,7 @@ void MapLoader::SetMapExplored(World& world)
         for(unsigned i = 0; i < MAX_PLAYERS; ++i)
         {
             // If we have FoW here, save it
-            if(world.GetNode(pt).fow[i].visibility == VIS_FOW)
+            if(world.GetNode(pt).fow[i].visibility == Visibility::FoW)
```

Comment:
we probalby should name `FoW` `FogOfWar` too

Reply:
I see no need to make it longer. Even the comment only uses `FoW` which in combination with `Visibility` is expressive enough, isn't it?

Reply:
but it would be more consistent, longer is not bad

Reply:
Good point. So I checked what we use so far: SaveFOWNode, fowVisibility, GetNode(pt).fow, GetYoungestFOWNode, isFoW, FoWNode, ...
So only the exploration enum uses the full name. Hence if we renamed this we'd need to rename all IMO, but I would rather not do that: GetYoungestFogOfWarNode looks overly verbose.
So I'd keep it as-is with the argument that `Exploration` is rather user facing so should not be abbreviated but everything else is internal and has enough context to make it clear what FoW means.

Or should we just change the enum so (only) the enums are consistent? Personally I like the shorter version better but no strong opinion for the enum

Reply:
I would make the enums consistent, the rest are bad, but okayish (one can still understand it) even if some of the other names are also a bit questionable (i.e isFow - shouldnt it be hasFow?) - the enums are more important I think (and one step by another)